llvm/utils/lit/tests/Inputs/shtest-env/env-args-none.txt

```diff
@@ -1 +1,30 @@
-# RUN: env
+# Check Default Environment
```

Comment:
```suggestion
# Check Default Environment.
```
Nit: comments should end with a '.'. Also, does this really need a capital 'd' and 'e'? Same throughout.

Reply:
I’ve already addressed that and made the necessary corrections. Thanks for the feedback!